chatgpt-retrieval-plugin/run.sh

```diff
@@ -22,7 +22,7 @@ $PYTHON_BINARY -m poetry install --with dev
 
 # Run tests. Sensitive variables in Evergreen come from Evergeen project: ai-ml-pipeline-testing/
 OPENAI_API_KEY=$openai_api_key \
```

Comment:
(In a different PR) Can we switch this to reference the local atlas URI?

Reply:
Yes! I started that work. Can't remember where I left it. 90% there. I'll make a ticket.